inc/css/custom.css

```diff
@@ -0,0 +1,396 @@
+/*
+--------------------------------------------------------------------------------
+Basic Elements
+*/
+/*<editor-fold desc="Basic Elements">*/
+input[type=submit] {
+    background-color: #e5bc6e;
+    border-radius: 0;
+    font-size: inherit;
+    color: white;
+    background-size: 200% 100%;
+    background-image: linear-gradient(to right, #e5bc6e 50%, #b3904d 50%);
+    transition: background-position 0.3s;
+    border: none;
+}
+ul {
+    margin: 0;
+}
+th {
+    color: #e5bc6e;
+}
+
+input[type=submit]:hover {
+    background-position: -100% 0;
+}
+input[type="text"], input[type="email"], input[type="url"], input[type="password"], 
+input[type="search"], input[type="number"], input[type="tel"], input[type="range"], 
+input[type="date"], input[type="month"], input[type="week"], input[type="time"], 
+input[type="datetime"], input[type="datetime-local"], input[type="color"], 
+textarea {
+    background-color: transparent;
+}
+/*</editor-fold>*/
+/*
+--------------------------------------------------------------------------------
+Header
+*/
+/*<editor-fold desc="Header">*/
+.header-icon {
+    margin: 10px;
+    width: 40px;
+    height: 40px;
+    background-color: #e5bc6e;
+    border-radius: 100%;
+    color: white;
+    line-height: 40px;
+    text-align: center;
+    vertical-align: middle;
+    display: inline-block;
+    transition: 0.4s ease-in-out;
+}
+#site-branding {
+    height: 70px;
+}
+#site-branding img {
+    max-height: 70px;
+    width: auto;      
+}
+#top-bar {
+    font-family: Trirong;
+    background-color: white;
+    box-shadow: 0px 0px 20px #737373;
+}
+#top-bar a {
```

Comment:
this impacts the site title ( when logo is not being used ).
Please consider adding a rule for the site title to set the color, otherwise it goes to white and it does not appear

![image](https://user-images.githubusercontent.com/3696057/28542741-524a3d78-708b-11e7-82c1-f0330498f8eb.png)